pkg/injector/config.go

```diff
@@ -105,6 +106,14 @@ func (c *Config) GetReadOnlyRootFilesystem() bool {
 	return utils.IsTruthy(c.ReadOnlyRootFilesystem)
 }
 
+func (c *Config) GetDropCapabilities() bool {
+	// Default is true if empty
+	if c.SidecarDropALLCapabilities == "" {
+		return true
```

Comment:
Should be `false`. This entire feature should be opt-in, otherwise we run the risk of breaking many systems in many places.